fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -2168,6 +2174,19 @@ public void setPreferComputeNode(boolean preferComputeNode) {
         this.preferComputeNode = preferComputeNode;
     }
 
+    public void setComputationFragmentSchedulingPolicy(String computationFragmentSchedulingPolicy) {
+        if (!"compute_nodes_only".equals(computationFragmentSchedulingPolicy)
```

Comment:
define string constant variable or enum type for `compute_nodes_only`/`all_nodes`. `computation_fragment_scheduling_policy` is not easy to understand, can think of a better name for it?

Reply:
the session variable name and value of computation_fragment_scheduling_policy,  is according to ZiheLiu(github ID) 's suggestion from previous pr(#38498) , could i adjust it ?

Reply:
ok then, make it constant variable or enum at least, don't hardcode it everywhere.